.github/workflows/cd.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     tags: 
       - '*'
+      - "daily-*"
```

Comment:
this looks like it's just adding `daily-*`, aren't suppose to have something indicating that it should be excluded, rather than included?

Reply:
I missed this when I pushed an update. This should be `!daily-*`